Makefile

```diff
@@ -50,8 +50,8 @@ test: prepare ## Run all go-sdk tests
 integration: build-cli-cross-platform integration-only ## Build and run integration tests
 
 .PHONY: integration-only
-integration-only: ## Run integration tests
-	PATH=$(PWD)/bin:${PATH} go test -v github.com/lacework/go-sdk/integration -timeout 30m -tags="\
+integration-only: install-tools ## Run integration tests
```

Comment:
why move install-tools under this command? CI already runs `make prepare` prior to run integration-only.

Reply:
I guess we need it now since this directive depends on `gotestsum` and if the developer doesn't have it installed, it will fail. I am ok with this.